<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDirDialog</class>
 <widget class="QDialog" name="ChangeDirDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Widget_bk_small" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_bk_small">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>原数据目录地址: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDatadir">
          <property name="text">
           <string>datadir</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>新数据地址: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditNewDataDir"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel { color: #ec5151; }</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ellipsisButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelFreeSpace">
          <property name="text">
           <string>(大约需要 60GB 空间)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errorMessage">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAttention">
          <property name="styleSheet">
           <string notr="true">QLabel { color: #ec5151; }</string>
          </property>
          <property name="text">
           <string>注意: 需要长时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>4</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>4</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Finish">
          <property name="text">
           <string>完成</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Change">
          <property name="text">
           <string>更改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Change</sender>
   <signal>clicked()</signal>
   <receiver>ChangeDirDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ChangeDirDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
